controllers/vrg_volrep.go

```diff
@@ -2018,20 +2018,32 @@ func (v *VRGInstance) validateExistingPV(pv *corev1.PersistentVolume) error {
 
 func (v *VRGInstance) validateExistingPVC(pvc *corev1.PersistentVolumeClaim) error {
 	existingPVC := &corev1.PersistentVolumeClaim{}
+	pvcNSName := types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}
 
-	err := v.reconciler.Get(v.ctx, types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}, existingPVC)
+	err := v.reconciler.Get(v.ctx, pvcNSName, existingPVC)
 	if err != nil {
-		return fmt.Errorf("failed to get existing PVC (%w)", err)
+		return fmt.Errorf("failed to get existing PVC %s (%w)", pvcNSName.String(), err)
 	}
 
-	if existingPVC.ObjectMeta.Annotations == nil ||
-		existingPVC.ObjectMeta.Annotations[RestoreAnnotation] != RestoredByRamen {
-		return fmt.Errorf("found PVC object not restored by Ramen for PVC %s", existingPVC.Name)
+	// If PVC not "Bound" return error
```

Comment:
The comment does not add much. Maybe we need a helper function so we can say:

    if !pvcs.IsBound(existingPVC) {